front/src/pages/auth/CreateProfile.tsx

```diff
@@ -54,6 +104,16 @@ export function CreateProfile() {
     [handleCreate],
   );
 
+  useEffect(() => {
+    if (currentUser?.firstName && currentUser?.lastName) {
```

Comment:
We seem to use this logic quite often. Should we create a `isUserOnbaorded` util fn ?